### PR TITLE
global: bump inspire-schemas to version ~33.0

### LINKED ISSUE #2267: general: bump inspire-schemas to version ~33.0

### PR DESCRIPTION
**Pull request by jacquerie**
## Related Issue
Closes #2267

## Checklist:
- [x] I have all the information that I need (if not, move to `RFC` and look for it).
- [x] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [x] I've added any new docs if API/utils methods were added.
- [x] I have updated the existing documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by jacquerie** [comment]
Missing `related_records2marc`, but meanwhile can you have a look, @michamos ?

**Comment by jacquerie** [comment]
Updated after our IRL discussion, @michamos. Care to take another look when you have time?

File: tests/unit/dojson/test_dojson_institutions.py [hunk 1]
```diff
@@ -884,38 +846,32 @@ def test_related_institutes_from__510_a_w_0_successor():
...
 
-    assert validate(result['related_institutes'], subschema) is None
-    assert expected == result['related_institutes']
+    assert 'related_records' not in result
 
 
-def test_related_institutes_from__invalid_510__0():
-    schema = load_schema('institutions')
-    subschema = schema['properties']['related_institutes']
+def test_related_records_from_510__a_w_0_discards_other():
```
Comment (michamos) [inline tests/unit/dojson/test_dojson_institutions.py]: this should be preserved, with `other` part of `relation_freetext`. The only reason for discarding relations (for now) is for relations going in the wrong direction, which require a solution to #2433.

**Comment by jacquerie** [comment]
Victim of #2358

**Pull request merged**